app/javascript/packages/document-capture/components/in-person-location-step.tsx

```diff
@@ -1,10 +1,13 @@
 import { useState, useEffect, useCallback, useRef, useContext } from 'react';
 import { useI18n } from '@18f/identity-react-i18n';
 import { PageHeading, SpinnerDots } from '@18f/identity-components';
+import { request } from '@18f/identity-request';
 import BackButton from './back-button';
 import LocationCollection from './location-collection';
 import LocationCollectionItem from './location-collection-item';
 import AnalyticsContext from '../context/analytics';
+import AddressSearch from './address-search';
+import AppContext from '../context/app';
 
 interface PostOffice {
```

Comment:
should this interface match the [postoffice struct](https://github.com/18F/identity-idp/blob/main/app/services/usps_in_person_proofing/post_office.rb)?

Reply:
the other struct has some other fields that aren't in this interface. I think we'd need those same fields here in order to display all the information that we display for each location on the location page.